src/commands/init.js

```diff
@@ -46,6 +46,16 @@ exports.handler = function ({ reporter, name, template }) {
   
   const basename = name.split('.')[0]
   const tasks = new TaskList([
+    {
+      title: 'Checking project existence',
+      task: async (ctx, task) => {
```

Comment:
`ctx` and `task` aren't really used here, but have been provided in other tasks that didn't use them as well. I just stuck to your style. Let me know if you want me to remove those.

Reply:
Lets keep them for consistency :)